task/03-date-tasks.js

```diff
@@ -56,10 +54,14 @@ function parseDataFromIso8601(value) {
  *    Date(2015,1,1)    => false
  */
 function isLeapYear(date) {
-   throw new Error('Not implemented');
+  if (
```

Comment:
`(date.getFullYear() % 4 === 0 && date.getFullYear() % 100 !== 0) || date.getFullYear() % 400 === 0` return logical result. You don't need to return `true/false` value directly.